en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+# Configuration/Tuning (Mecanum Rover)
+
+This topic provides a step-by-step guide for setting up your [Mecanum rover](../frames_rover/mecanum.md).
+Successive steps enable [drive modes](../flight_modes_rover/mecanum.md) with more autopilot support and features.
+
+::: warning
+Each step is dependent on the previous steps having been completed.
+Modes will only work properly if the preceding modes have been configured.
+:::
+
+## Basic Setup
+
+To start using the mecanum rover:
+
+1. Enable Rover support by flashing the [PX4 rover build](../frames_rover/index.md#flashing-the-rover-build) onto your flight controller.
+   Note that this is a special build that contains rover-specific modules.
+
+2. In the [Airframe](../config/airframe.md) configuration select _Generic Rover Mecanum_ frame:
+
+   ![QGC screenshot showing selection of the airframe 'Generic Rover Mecanum'](../../assets/config/airframe/airframe_generic_rover_mecanum.png)
+
+   Select the **Apply and Restart** button.
+
+   ::: info
+   If this airframe does not show up in the UI, it can alternatively be selected by setting the [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) parameter to `52000`.
+   :::
+
+3. Use [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
+
+## Manual Mode
+
+The basic setup (above) is all that is required to use the rover in [Manual mode](../flight_modes_rover/mecanum.md#manual-mode).
+
+::: info
+In manual mode the stick inputs are directly mapped to motor commands.
+Especially moving the stick that controls the yaw rate all the way to one side will cause the wheels on the left and right to spin at full speed in opposite directions.
+Depending on the rover this can lead to a very aggressive rotation.
+The parameter [RM_MAN_YAW_SCALE](#RM_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
+By reducing the parameter from the default value of 1 this behaviour can be tuned.
+Note that this parameter only affects this mode, not any of the following ones.
+:::
+
+## Acro Mode
+
+To set up [Acro mode](../flight_modes_rover/mecanum.md#acro-mode) navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
+
+1. [RM_WHEEL_TRACK](#RM_WHEEL_TRACK) [m]: Measure the distance from the centre of the right wheel to the centre of the left wheel.
+
+   ![Wheel track](../../assets/airframes/rover/rover_differential/wheel_track.png)
+
+2. [RM_MAX_YAW_RATE](#RM_MAX_YAW_RATE) [deg/s]: This is the maximum yaw rate you want to allow for your rover.
```

Comment:
Is there advice for a good initial value? What happens if you get it wrong?

Reply:
Elaborated on it in https://github.com/PX4/PX4-user_guide/pull/3411/commits/6474c87f6d8345975b93dafd472d8e9d18fbf632

Reply:
Nice! Subedited (and ran prettier) in https://github.com/PX4/PX4-user_guide/pull/3411/commits/07a21f52161798ec4aba89dc72987d911959b0c6